nimbleclient/v1/restclient.py

```diff
@@ -203,30 +228,40 @@ def get(self, endpoint, **params):
             paginated_data.extend(response.json()["data"])
             total_rows = response.json()['totalRows']
             retrieved_rows = len(paginated_data)
-            #current_end_row = response.json()['endRow']
+            # current_end_row = response.json()['endRow']
 
             if 'startRow' in params and 'endRow' in params:
```

Comment:
The original intent of grouping parameters into `params` is to work on those fields as a group. If you're expecting `startRow` and `endRow` as valid function parameters that you're not going to use as a group, then you should add them into the function signature and use `snake_case` so its consistent with the rest of the code: `def get_data(self, endpoint, filter=None, start_row=None, end_row=None, **params):`.

I know this affects some of your other logic, but it's better to be explicit than implicit.

Reply:
Agreed, refactored the get() method and fixed accordingly.